src/main/scala/edu/cmu/cs/obsidian/codegen/yulAST.scala

```diff
@@ -276,8 +278,8 @@ case class YulObject(name: String, code: Code, subobjects: Seq[YulObject], data:
                 codegen.ExpressionStatement(apply("abi_decode_tuple", intlit(4), apply("calldatasize"))),
                 //    fun_retrieve_24()
                 call_f_and_maybe_assign,
-                //    let memPos := allocate_memory(0)
-                decl_1exp(mp_id, apply("allocate_memory", intlit(0))),
+                //    let memPos := allocate_unbounded()
```

Comment:
What does allocate_unbounded do?

Reply:
it's a less complicated version of allocation that the more recent build of solc outputs; i shifted to it while trying to debug the `0x` return from Ganache that turned out to be something else. i think either way is OK for now, but figuring out how to do this right is part of #333